database-service/tests/api/conftest.py

```diff
@@ -0,0 +1,71 @@
+from utils import json_helpers
```

Comment:
shouldn't this be `from test_utils` ?

Reply:
Would that import fail, the whole tests would fail; therefore, the import statement is correct. You need to double-check the mappings in the docker-compose. That might be improved in a later phase.